start-points/12_Days_of_Xmas/readme.txt

```diff
@@ -114,7 +114,7 @@ Six geese a-laying
 Five golden rings
 Four calling birds
 Three french hens
-Two turtle doves
-And a partridge in a pear tree.
```

Comment:
Inconsistent with the rest of the song.
This is actually the way I would prefer it, because this is how the breaks fall when it's sung. But I decided to limit the scope of this PR.